content/id/docs/tasks/tools/install-kubectl.md

```diff
@@ -0,0 +1,496 @@
+---
+title: Instalasi dan Konfigurasi kubectl
+content_template: templates/task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Instalasi kubectl
+---
+
+{{% capture overview %}}
+[Kubectl](/docs/user-guide/kubectl/) adalah perangkat _command line_ Kubernetes yang digunakan untuk menjalankan perintah-perintah untuk kluster Kubernetes. Kamu dapat menggunakan `kubectl` untuk men-_deploy_ aplikasi, mengatur _resource_ kluster, dan melihat _log_. Daftar operasi `kubectl` dapat dilihat di [Ikhtisar kubectl](/docs/reference/kubectl/overview/).
```

Comment:
how about translating command line to "barisan perintah"?

Reply:
I think "berbagai perintah" sounds better. wdyt @girikuncoro ?

Reply:
I think barisan perintah (command-line) is also used in other translations: https://git-scm.com/book/ms/v2/Untuk-Bermula-Barisan-Perintah-ataupun-Command-Line

Reply:
Oh I'm sorry, I got the wrong highlight, I thought the one you mention was to replace "perintah-perintah", turns out the "_command line_" one.
Yeah, "barisan perintah" sounds good, I'll change the others too. Thanks @girikuncoro!